src/field/captcha/captcha_pane.jsx

```diff
@@ -7,22 +7,38 @@ import * as l from '../../core/index';
 import { swap, updateEntity } from '../../store/index';
 import * as captchaField from '../captcha';
 import { getFieldValue, isFieldVisiblyInvalid } from '../index';
-import RecaptchaV2 from './recaptchav2';
+import { ReCAPTCHA } from './recaptchav2';
 
 export default class CaptchaPane extends React.Component {
   render() {
     const { i18n, lock, onReload } = this.props;
 
     const lockId = l.id(lock);
 
-    function handleChange(e) {
-      swap(updateEntity, 'lock', lockId, captchaField.set, e.target.value);
-    }
-
     const captcha = l.captcha(lock);
 
     if (captcha.get('provider') === 'recaptcha_v2') {
-      return <RecaptchaV2 lock={lock} siteKey={captcha.get('siteKey')} />;
+      function handleChange(value) {
+        swap(updateEntity, 'lock', lockId, captchaField.set, value);
+      }
+
+      function reset() {
+        handleChange();
+      }
+
+      return (
+        <ReCAPTCHA
+          sitekey={captcha.get('siteKey')}
+          onChange={handleChange}
+          onExpired={reset}
+          hl={l.ui.language(lock)}
```

Comment:
Here we're taking it from Lock options. I wonder if the line with the fixed language I pointed in my other comment has any effect on this.

Reply:
The other comment is in a line inside a unit testing where I am testing that the recaptcha component links to the right url in google. Then in this line we are instantiating the recaptcha component with the language that was used in Lock. I've tested this manually and it works as expected, if lock is configured in spanish then recaptcha is in spanish, if lock is in english (or the lang is not set) then recaptcha is in English. 

If you don't specify the language to google recaptcha it will default to the language of your browser, which might lead to a broken ui. So I tested that we always send the language to google

Reply:
Thanks. I completely missed the other was part of a unit test. 🤕